extension/js/agent/utils/knownTypes.js

```diff
@@ -183,17 +183,29 @@
       });
     }
 
-
+    //Marionette 2.0.0 does not have Radio in Wreqr
     if (Agent.patchedBackbone.Wreqr) {
-      _.extend(knownTypes, {
-        'Backbone.Wreqr.Channel': {
-          type: Agent.patchedBackbone.Wreqr.Channel,
-          name: 'wreqr-channel',
-          className: 'Wreqr.Channel',
-          cid: function(obj) { return ''},
-          toString: toString
-        }
-      });
+      if(Agent.patchedBackbone.Wreqr.Channel) {
+        _.extend(knownTypes, {
+          'Backbone.Wreqr.Channel': {
+            type: Agent.patchedBackbone.Wreqr.Channel,
+            name: 'wreqr-channel',
+            className: 'Wreqr.Channel',
+            cid: function(obj) { return ''},
+            toString: toString
+          }
+        });
+      } else {
+        _.extend(knownTypes, {
+          'Backbone.Wreqr': {
+            type: Agent.patchedBackbone.Wreqr.__proto__.constructor,
```

Comment:
Because the instance of check that is performed later down the line on the 'type' property expects a function.  And Wreqr has no public constructor as it were, its a literal object / singleton.  So the only way I could see to get a function in there to check in the instance of was to hit the constructor.